### ISSUE TITLE
Build Clubs Curriculum page

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.webmaker.org#42]
### RACI
* Phase: Build
* Owner: @hannahkane 
* Decision Maker: @hannahkane 
* Design Lead: NA
* Dev Lead: 
* Quality: @davidascher 

Design assets will come from #106.

### Requirements
* Responsive
* Works in all supported browser versions
* Does NOT need to be dynamic. The curriculum titles/images/blurbs can be static for v1.
* Does NOT need to be l10n'ed for v1. Any translated curriculum can be linked to from the github pages themselves.

Tasks:
 - [ ] Design review
 - [ ] Content review

**Comment by hannahkane** [comment]
This ticket is now unblocked. See #106.

**Comment by gesa** [comment]
@vazquez could I get all the assets for this please?

**Comment by vazquez** [comment]
Give me 5 minutes!

**Comment by vazquez** [comment]
@gesa Clubs Curriculum Assets available here: https://dl.dropboxusercontent.com/u/43783651/Mozilla/Clubs%20Curriculum%20Assets.zip

**Comment by gesa** [comment]
@vazquez thank you so much! A couple of requests:
- could the vector graphics be re-exported as SVG?
- could the hero image be re-exported as a square? (and it might be more light-weight if it was a jpg? maybe? not totally sure.)

Or if it's easier for you, I'd happily do the exporting from a Sketch file.

**Comment by vazquez** [comment]
Yeah I had doubts on what the best way to export the hero image, since the circle can be done in CSS. No problem! I will re-export the hero image as a square. That way it can be a JPG too :)

Regarding the vector graphics being re-exported as SVG, which ones do you mean exactly? Do you mean the `icon-web-literacy.jpg`?

**Comment by vazquez** [comment]
Hero images have been re-exported as squares and are now JPG: https://dl.dropboxusercontent.com/u/43783651/mozilla/Clubs%20Curriculum%20Assets.zip

**Comment by gesa** [comment]
Sorry I didn't get back to you immediately @vazquez.

So I don't know the source of these icons (if you designed them yourself or they were given to you as assets) but I'm p sure these all could be svg:
```
img/clubs-curriculum-page
|-- icon-web-literacy.jpg
|-- img-hack-news.jpg
|-- img-kraken-code.jpg
|-- img-ping-kong.jpg
|-- img-story-of-us.jpg
`-- img-web-chef.jpg
```

But *especially* `icon-web-literacy.jpg`

**Comment by vazquez** [comment]
Sadly none of those can be SVG as they are images that we took from the current page. We will recreate those assets in SVG eventually, but not for now as it is time consuming.

`icon-web-literacy.jpg` needs to be a JPG because were using some radial gradients *within* a shape *as* a border, which SVG does not understand well. I learned this lesson in the About page:

![screenshot 2015-03-20 17 26 25](https://cloud.githubusercontent.com/assets/1119821/6761139/4b31df18-cf26-11e4-9e06-15a7c078a63c.png)
